rgen/rgen-viewer/src/main.rs

```diff
@@ -184,8 +215,8 @@ pub fn main() -> Result<(), String> {
       f.render(0, 0, format!("X: {x:0.2} Z: {z:0.2}", x = hover_pos.x, z = hover_pos.z));
       f.render(0, 24, format!("Height: {meter_height:0.2}"));
 
-      let biome = world.biome_at(hover_pos);
-      f.render(0, 48, format!("Biome: {}", world.context.biomes.name_of(biome)));
+      //let biome = world.biome_at(hover_pos);
+      //f.render(0, 48, format!("Biome: {}", world.context.biomes.name_of(biome)));
```

Comment:
why'd you comment this out?

Reply:
It was throwing up an error

Reply:
apparently there was another error. going to figure it out later